include/picolibrary/indicator.h

```diff
@@ -36,6 +40,67 @@ enum class Initial_Indicator_State {
     EXTINGUISHED, ///< Extinguished.
 };
 
+/**
+ * \brief Fixed intensity indicator concept.
+ */
+class Fixed_Intensity_Indicator_Concept {
+  public:
+    Fixed_Intensity_Indicator_Concept() = delete;
+
+    Fixed_Intensity_Indicator_Concept( Fixed_Intensity_Indicator_Concept && ) = delete;
+
+    Fixed_Intensity_Indicator_Concept( Fixed_Intensity_Indicator_Concept const & ) = delete;
+
+    ~Fixed_Intensity_Indicator_Concept() = delete;
+
+    auto operator=( Fixed_Intensity_Indicator_Concept && ) = delete;
+
+    auto operator=( Fixed_Intensity_Indicator_Concept const & ) = delete;
+
+    /**
+     * \brief Initialize the indicator's hardware.
+     *
+     * \param[in] initial_indicator_state The initial state of the indicator.
+     *
+     * \return Nothing if indicator hardware initialization succeeded.
+     * \return An error code if indicator hardware initialization failed. If indicator
+     *         hardware initialization cannot fail, return
+     *         picolibrary::Result<picolibrary::Void, picolibrary::Void>.
+     */
+    auto initialize( Initial_Indicator_State initial_state = Initial_Indicator_State::EXTINGUISHED ) noexcept
+        -> Result<Void, Error_Code>;
+
+    /**
+     * \brief Illuminate the indicator.
+     *
+     * \return Nothing if illuminating the indicator succeeded.
+     * \return An error code if illuminating the indicator failed. If illuminating the
+     *         indicator cannot fail, return picolibrary::Result<picolibrary::Void,
+     *         picolibrary::Void>.
+     */
+    auto illuminate() noexcept -> Result<Void, Error_Code>;
+
+    /**
+     * \brief Extinguish the indicator.
+     *
+     * \return Nothing if extinguishing the indicator succeeded.
+     * \return An error code if extinguishing the indicator failed. If extinguishing the
+     *         indicator cannot fail, return picolibrary::Result<picolibrary::Void,
+     *         picolibrary::Void>.
+     */
+    auto extinguish() noexcept -> Result<Void, Error_Code>;
+
+    /**
+     * \brief Toggle the indicator.
+     *
+     * \return Nothing if toggling the indicator succeeded.
+     * \return An error code if toggling the indicator failed. If toggling the indicator
+     *          cannot fail, return picolibrary::Result<picolibrary::Void,
+     *          picolibrary::Void>.
```

Comment:
Please fix indentation of subsequent comment lines.

```suggestion
     * \return An error code if toggling the indicator failed. If toggling the indicator
     *         cannot fail, return picolibrary::Result<picolibrary::Void,
     *         picolibrary::Void>.
```